src/components/toolbox/table/table.css

```diff
@@ -97,3 +97,35 @@
 .sortAsc::after {
   transform: rotate(180deg);
 }
+
+.titleWrapper {
+  position: relative;
+}
+
+.sortInactive {
```

Comment:
This class replaces `sortable` on line 79 in this file, so it can be removed there.

I would comment directly on line 79, but I cannot as it wasn't changed.

Reply:
✅